r/adbcdrivermanager/DESCRIPTION

```diff
@@ -0,0 +1,27 @@
+Package: adbcdrivermanager
+Title: 'Arrow' Database Connectivity ('ADBC') Driver Manager
```

Comment:
There is a trick to get rid of the quotes. @maelle: Can you please help?

Reply:
https://ropensci.org/blog/2022/01/21/ropensci-news-digest-january-2022/#to-quote-or-not-to-quote-non-existing-words-in-description (please tell me if I need to make an actual PR :smile_cat: )